### PR TITLE
Create basic UI

### PR DESCRIPTION
**Pull request by edwardoluyara**
- Create a `main.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
 ```Ruby
#!/usr/bin/env ruby
require './app'

def main
  app = App.new
  app.run
end

main
```

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    @persons = []
+    @books = []
+    puts "\nWelcome to the School Library App!"
+    display_menu
+  end
+
+  def display_menu
+    puts "\nPlease choose an option by entering a number:"
+    puts "\n1 - List all books"
+    puts '2 - List all people'
```
Comment (er-contreras) [inline app.rb:17]: - [ ] When listing all people it shows a wrong name in student and a wrong name and specialization from the teacher.

Screen Shot 2022-03-11 at 11 21 02

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```
Comment (er-contreras) [inline app.rb:20]: - [ ] When I type 'Create a rental' it doesn't throw the next questions to create it.

Screen Shot 2022-03-11 at 11 24 13

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (edwardoluyara) [reply app.rb:20]: - Updated `list_all_people` now it looks like this:

  - listOfAllPersons

- Updated `create_a_rental` now it looks like this:

  - createdRental

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+  def list_all_rentals
+    print 'ID of person: '
+    id = gets.chomp.to_i
+
+    puts 'Rentals:'
+    @rentals.each do |rental|
+      puts "Date: #{rental.date}, Book '#{book.title}' by #{book.author}" if rental.person == id
+    end
+    sleep 0.75
```
Comment (elisha2kyakpo1) [inline app.rb]: - [ ] Let's work on this function when I create a Rental, I get a successful message that Rental was created, but when I want to see the list of created rentals I see nothing. See the image below

rental

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

- The "list all people" was worked on very well and it's working 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 3 above)
Comment (edwardoluyara) [reply app.rb]: - Hello @elisha2kyakpo1 I have finally fixed the bug

  - listAllRentals

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @oluyaratosin123, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**